client/additional-methods-setup/methods-selector/setup-complete-task.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useEffect, useCallback, useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { getHistory, getNewPath } from '@woocommerce/navigation';
+import { useDispatch } from '@wordpress/data';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import CollapsibleBody from '../wizard/collapsible-body';
+import WizardTaskItem from '../wizard/task-item';
+import WizardTaskContext from '../wizard/task/context';
+import './setup-complete-task.scss';
+
+const SetupComplete = () => {
+	const { isActive } = useContext( WizardTaskContext );
+
+	const handleGoHome = useCallback( () => {
+		getHistory().push( getNewPath( {}, '/', {} ) );
+	}, [] );
+
+	const { updateOptions } = useDispatch( 'wc/admin/options' );
+	const { createInfoNotice } = useDispatch( 'core/notices' );
+
+	useEffect( () => {
+		if ( ! isActive ) {
+			return;
+		}
+
+		createInfoNotice(
+			__(
+				'This content will be updated in a subsequent PR. Well done!',
+				'woocommerce-services'
+			)
+		);
+
+		updateOptions( {
+			// eslint-disable-next-line camelcase
+			wcpay_additional_methods_setup_completed: 'yes',
+		} );
+
+		// Set the local `isSetupCompleted` to `yes` so that task appears completed on the list.
+		// Please note that marking an item as "completed" is different from "dismissing" it.
+		window.wcpayAdditionalMethodsSetup.isSetupCompleted = 'yes';
```

Comment:
Once the whole task is completed, it is marked as completed and there is a visual indication.
<img width="639" alt="CompletedTask" src="https://user-images.githubusercontent.com/8002881/118757064-e207c380-b831-11eb-8abe-ffedb3700869.png">
I don't think (or missed) we are adding this information to the context or passing it down to the `WizardTask` 

After you have completed the whole task, if you click again on it, the individual steps (WizardTasks) appear as incomplete.
<img width="832" alt="StepsWizardTasks" src="https://user-images.githubusercontent.com/8002881/118757531-e5e81580-b832-11eb-9cc0-4936eed026c4.png">

Is this by design? 
Other completed tasks in WC, like the store setup, work differently. They will show that you completed the steps.